packages/nimble-components/src/table/template.ts

```diff
@@ -83,8 +83,14 @@ export const template = html<Table>`
                             ${repeat(x => x.visibleColumns, html<TableColumn, Table>`
                                 <div class="header-container">
                                     ${when((_, c) => c.index > 0, html<TableColumn, Table>`
-                                        <div class="column-divider left ${(_, c) => `${c.parent.layoutManager.activeColumnIndex === c.index ? 'active' : ''}`}" 
-                                             @mousedown="${(_, c) => c.parent.onLeftDividerMouseDown(c.event as MouseEvent, c.index)}">
+                                        <div
+                                            class="
+                                                column-divider
+                                                left
+                                                ${(_, c) => `${c.parent.layoutManager.activeColumnIndex === c.index ? 'visible' : ''}`}
```

Comment:
So a divider can be visible without having the `visible` class, right? It looks like `visible` is only set when a divider is being dragged. That initially confused me. Could we change the class to something like `column-sizing` or `column-active`?

Reply:
That's a good point. I've updated the two classes to be `column-active` and `divider-active`.